packages/deployment/app/workflow_lambda_versions.config.yml

```diff
@@ -1,8 +1,12 @@
 default:
+
+  override_with_parent:
```

Comment:
I see that you're not merging into master -- is this going to be documented?

Reply:
Hmm, good point.   This should be updated to be merged *to* master once the main 885 PR is merged, I seperated it out for review clarity, but the base branch was not master initially.    I'd be happy to reissue the PR against master if that's of concern, else I'll modify the target once the other PR is resolved. 

In either case, documenting these internal features is of some concern - the correct implementation in this case ( absent other concerns ) should probably to involve a PR back to kes proper rather than an override.      

There's also a story [CUMULUS-870] that addresses this concern from 850 as well.

Reply:
@laurenfrederick  068a1761